src/platform_specific/affinity.unix.cc

```diff
@@ -0,0 +1,23 @@
+
+#ifdef __linux__
+
+#include <cassert>
+#include <pthread.h>
+#include <sched.h>
+#include <unistd.h>
+
+typedef cpu_set_t native_cpu_set;
```

Comment:
1) This alias seems rather pointless, since `cpu_set_t` is just as good a name as `native_cpu_set` and this type is only needed in the function below.
2) I don't think `typedef` should be used at all in modern C++ since we have `using` now.